jenkins/ci.opensuse.org/openstack-trackupstream.yaml

```diff
@@ -95,6 +96,7 @@
             "openstack-monasca-log-api",
             "openstack-monasca-notification",
             "openstack-monasca-persister",
+            "openstack-monasca-transform",
```

Comment:
I think it sh ould be in the block line 116 (it only exists pike/master)

Reply:
Is that true for all the monasca excludes?

Reply:
no, I think th eother monasca packages were backported properly because of SOC7 Monitoring release. this package is a new pike+ only thing